protocol/app/testdata/default_genesis_state.json

```diff
@@ -454,6 +454,7 @@
     "total_escrowed": []
   },
   "upgrade": {},
+  "vault": {},
```

Comment:
do we even need to add this to genesis? It seems like at genesis on mainnet/testnet, we didn't have a vault, so wouldn't we default to `{}` for vault?

Reply:
this is needed for tests on default genesis state. added a new module so genesis state has this new entry